src/NzbDrone.Core/Indexers/Newznab/NewznabRequestGenerator.cs

```diff
@@ -57,6 +57,18 @@ private bool SupportsTvdbSearch
                        capabilities.SupportedTvSearchParameters.Contains("ep");
             }
         }
+        private bool SupportsImdbSearch
+        {
+            get
+            {
+                var capabilities = _capabilitiesProvider.GetCapabilities(Settings);
+
+                return capabilities.SupportedTvSearchParameters != null &&
+                       capabilities.SupportedTvSearchParameters.Contains("imdbid") &&
```

Comment:
Do indexers actually support this parameter?

Reply:
Yes!
At least Jackett support only this:
https://github.com/Jackett/Jackett/wiki/Definition-format

"Modes: # Specify which torznab search modes and attributes are supported by the
           # indexers.
           # Implementation note: Jackett doesn't care very much about this but you
           # still should specify the correct modes.
           # Especially don't include imdbid if the tracker doesn't support
           # searching by imdbid.
      search: [q, imdbid]
      **tv-search: [q, season, ep, imdbid]**
      movie-search: [q, imdbid]"